src/Psl/Type/nonnull.php

```diff
@@ -17,8 +15,8 @@
  */
 function nonnull(): TypeInterface
```

Comment:
```suggestion
function nonnull(): NonNull
```

shouldn't this be updated?